media_softlet/linux/common/os/osservice/mos_utilities_specific.h

```diff
@@ -47,7 +47,8 @@ class MosMutex;
 
 //user feature
 #if ANDROID_VERSION >= 800
-#define USER_FEATURE_FILE                   "/data/igfx_user_feature.txt"
+#define USER_FEATURE_FILE                   "/vendor/etc/igfx_user_feature.txt"
+#define USER_FEATURE_FILE_NEXT              "/vendor/etc/igfx_user_feature_next.txt"
```

Comment:
suggest change to
#ifdef ANDROID
We don't need support version before Android 8.
Also if we need handle separately for Android version, should use __ANDROID_API__.